airbyte-integrations/connectors/source-scaffold-java-jdbc/src/test-integration/java/io/airbyte/integrations/source/scaffold_java_jdbc/ScaffoldJavaJdbcSourceAcceptanceTest.java

```diff
@@ -39,7 +40,7 @@
   private JsonNode config;
 
   @Override
-  protected void setup(TestDestinationEnv testEnv) throws Exception {
+  protected void setupEnvironment(TestDestinationEnv testEnv) {
```

Comment:
@mmolimar any reason to change the name of this function?

Reply:
The ``setup`` method doesn't exist in its super class ``SourceAcceptanceTest`` so it didn't compile but it does the ``setupEnvironment``.